docs/nightly/en/user-guide/continuous-aggregation/expression.md

```diff
@@ -1,9 +1,101 @@
 # Expression
 
-This part list all supported aggregate functions.
+## aggregate functions
+
+This part list all supported aggregate functions in flow.
 
 - `count(column)`: count the number of rows.
 - `sum(column)`: sum the values of the column.
 - `avg(column)`: calculate the average value of the column.
 - `min(column)`: find the minimum value of the column.
 - `max(column)`: find the maximum value of the column.
+
+## scalar functions
+
+Flow support most scalar functions from datafusion, see the full list of supported scalar functions in [datafusion](https://datafusion.apache.org/user-guide/sql/scalar_functions.html#scalar-functions).
+
+And here are some of the most commonly used scalar functions in flow:
```

Comment:
No need to write the same content twice. We can delete the following content.

Reply:
We can link to the https://docs.greptime.com/reference/sql/functions/df-functions#date-bin 

A list to describe these important functions could be enough.